gobblin-runtime/src/main/java/org/apache/gobblin/runtime/mapreduce/MRJobLauncher.java

```diff
@@ -262,6 +265,10 @@ protected void runWorkUnits(List<WorkUnit> workUnits) throws Exception {
     JobState jobState = this.jobContext.getJobState();
 
     try {
+      CountEventBuilder countEventBuilder = new CountEventBuilder(JobEvent.WORK_UNITS_CREATED, workUnits.size());
```

Comment:
Why not emit WorkUnitsCreated for Local and Helix jobs as well?

Reply:
So we are no longer emitting workUnitsCreated timer for LocalJobLauncher?

Reply:
We are workUnitsCreatedTimer events as well. That has not changed for local job launcher.